{{cookiecutter.directory_name}}/pyproject.toml

```diff
@@ -99,6 +94,35 @@ extras = dev
 """
 
 [tool.ruff]
+exclude = [
```

Comment:
Is this really needed? Can't ruff just use the list from .gitignore?

Reply:
These exclusions are part of the default configuration, so don't need to be specified. In fact, by leaving it empty, the exclusions will stay up to date with whatever the default is rather than what is defined here.

Reply:
I'm deferring this to #397 and marking this as resolved